.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Description
+[Describe what this change achieves]
+
+### Issues Resolved
+[List any issues this PR will resolve]
+
+### Check List
+- [ ] New functionality includes testing.
+  - [ ] All tests pass
```

Comment:
Let's make periods at the end of sentences consistent here in the list. Add them to the lines missing them.